src/components/PostListing.js

```diff
@@ -163,18 +161,19 @@ const PostListing = React.createClass({
   render: function() {
     return (
       <View style={styles.outerListView}>
-        <RefreshableListView
-          ref = {(listView) => {this.listView = listView;}}
-          renderScrollComponent={props => <InfiniteScrollView {...props} />}
+        <ListView
+          ref="listView"
           dataSource={this.state.dataSource}
           renderRow={this.renderRow}
-          loadData={this.props.refresh}
           automaticallyAdjustContentInsets={false}
-          distanceToLoadMore={200}
-          refreshDescription="Refreshing articles"
           style={styles.listView}
-          onLoadMoreAsync={this.props.onLoadMoreAsync}
-          canLoadMore
+          refreshControl={
+            <RefreshControl
+              onRefresh={this.props.refresh}
+              refreshing={this.state.refreshing}
+              title="Refreshing articles" />
+          }
+          onEndReached={this.props.onLoadMoreAsync}
```

Comment:
Any way we can have this happen a particular distance from the bottom as we did prior in line 173? I think it would have something to do with [this prop](https://facebook.github.io/react-native/docs/listview.html#onendreachedthreshold). I think this would set the threshold before calling `onEndReached`.